src/util/generatePreview.js

```diff
@@ -39,6 +39,63 @@ const errorHandlerScript = `(${function() {
   };
 }.toString()}());`;
 
+const elementHighlighterScript = `(${function() {
+  let selector = '';
+  window.addEventListener('message', (message) => {
+    const data = JSON.parse(message.data);
+    const {selector: {selector: selectorFromMessage}} = data;
+    selector = selectorFromMessage;
+    removeCovers();
+    generateCovers();
+  });
+
+  window.addEventListener('resize', () => {
+    removeCovers();
+    generateCovers();
+  });
+
+  window.addEventListener('scroll', () => {
```

Comment:
What’s the thinking here? Attaching basically _anything_ to a scroll handler without throttling/debouncing is a recipe for performance problems.

Reply:
Okay! I want to update the cover's position when the user scrolls or resizes the window since its location is based off its position relative to the frame.  There was no thinking here,  I hadn't used throttling or debouncing before. Looking into how to incoperate throttling and debouncing.

Reply:
How would you suggest importing lodash into the preview frame?

Reply:
I would probably just hand-roll a throttle function, at least until we do [this](https://github.com/popcodeorg/popcode/pull/1025#pullrequestreview-62624580). That said, it’s not clear to me why we need to handle scroll at all—resize makes sense but given that the highlight boxes are absolutely positioned, why would they need to be recalculated on scroll?